plugin.xml

```diff
@@ -34,7 +34,7 @@
         <engine name="cordova-android" version=">=6.3.0" />
```

Comment:
You need to also change this:

```xml
<engine name="cordova-android" version=">=10.0.0" />
```

The file plugin `7.0.0` implemented the `WebViewAssetLoader` proxy handler to try and continue to support something like the `cdvfile` process but in a new manner. The `WebViewAssetLoader` is a part of the `AndroidX` library and cordova-android 10.x is when `AndroidX` was introduced.

If this is not updated, there will be build failures because the older platform versions does not support the latest libraries.

Reply:
This required `engine` cordova-android v10 for plugin-file v7 is correct, but is already declared in plugin-file itself [here](https://github.com/apache/cordova-plugin-file/blob/7.0.x/plugin.xml#L33). The `engine` value here for plugin-media is not supposed to be linked the the one needed by its required plugins, but by its own code I think. Again there was no code change in this plugin that rely on cordova-android APIs newer than current `>=6.3.0`.

Reply:
Done